testdata/go/v4-alpha/monitoring/memcached-operator/Dockerfile

```diff
@@ -12,17 +12,17 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
+COPY cmd/main.go cmd/main.go
 COPY api/ api/
-COPY controllers/ controllers/
+COPY internal/controller/ internal/controller/
 COPY monitoring/ monitoring/
```

Comment:
We would have to look into if we can rip of certain parts of monitoring/ and put it with internal/controller/. I assume the most of the registration of metrics would be done in the controller. This investigation is just for follow up not this PR.